.github/workflows/test_iree.yml

```
@@ -140,13 +139,16 @@ jobs:
         run: |
           source ${VENV_DIR}/bin/activate
           python3 -m pip install -r iree_tests/requirements.txt
+          python3 -m pip install -r iree_special_models/requirements.txt
+          pip install --no-compile --pre --upgrade -e common_tools
 
       # Download remote files.
       - name: "Downloading remote files for real weight model tests"
         run: |
           source ${VENV_DIR}/bin/activate
-          python3 iree_tests/download_remote_files.py --root-dir pytorch/models
-          python3 iree_tests/download_remote_files.py --root-dir sharktank
+          python3 iree_tests/download_remote_files.py --root-dir iree_tests/pytorch/models
+          python3 iree_tests/download_remote_files.py --root-dir iree_special_models
+          python3 iree_tests/download_remote_files.py --root-dir iree_tests/sharktank
```

nit: sort (could put special models at the top for alphabetical, or put lower)

```suggestion
          python3 iree_tests/download_remote_files.py --root-dir iree_special_models
          python3 iree_tests/download_remote_files.py --root-dir iree_tests/pytorch/models
          python3 iree_tests/download_remote_files.py --root-dir iree_tests/sharktank
```

We may also want to make the cache scanning faster as we add more files